packages/vite/src/node/cli.ts

```diff
@@ -217,7 +217,7 @@ cli
             }
           },
           'serve',
```

Comment:
Should this line be changed to `build` too? Because we don't actually need vite server plugins in `vite preview`.